Packs/CortexXDR/ReleaseNotes/4_8_16.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Cortex XDR Malware - Incident Enrichment
+Added Account Enrichment playbook.
```

Comment:
The playbook now supports account enrichment.

Reply:
Done